specification/communication/data-plane/Email/preview/2023-01-15-preview/CommunicationServicesEmail.json

```diff
@@ -0,0 +1,384 @@
+          {
+            "in": "header",
+            "name": "Operation-Id",
+            "description": "This is the ID used by the status monitor for this long running operation.",
+            "type": "string",
+            "required": false,
+            "format": "uuid"
+          },
+          {
+            "in": "header",
+            "name": "x-ms-client-request-id",
+            "description": "Tracking ID sent with the request to help with debugging.",
+            "type": "string",
+            "required": false,
+            "format": "uuid",
+            "x-ms-client-name": "ClientRequestId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "emailMessage",
+            "description": "Message payload for sending an email",
+            "required": true,
+            "x-ms-client-name": "message",
+            "schema": {
+              "$ref": "#/definitions/EmailMessage"
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "headers": {
+              "x-ms-error-code": {
+                "description": "Error code - this will be the same as the code in the error property in the response body.",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorResponse"
+            }
+          },
+          "202": {
+            "description": "The service has accepted the request and will start processing the message later. It will return 'Accepted' immediately and include an 'Operation-Location' header. Client side should further query the operation/message status using the URL specified in 'Operation-Location' header. Once the send operation has succeeded, you can get additional status about email delivery through event grid events (for events reference, please refer to: https://learn.microsoft.com/en-us/azure/event-grid/communication-services-email-events)",
+            "headers": {
+              "Operation-Location": {
+                "description": "Location url of where to poll the status of this operation from.",
+                "type": "string"
+              },
+              "retry-after": {
+                "description": "This header will only be present when the operation status is a non-terminal status. It indicates the minimum amount of time in seconds to wait before polling for operation status again.",
+                "type": "integer",
+                "format": "int32"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/EmailSendResult"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
```

Comment:
nit, are you sure about the reason you write this `"final-state-via": "azure-async-operation"`?

Reply:
yes, I went through the documentation for long running operation and I believe this is the right thing to use. We have samples using the generated SDK and that seems to be working fine for our use-case. We don't have a reason to make a call to a final GET at the end of our POST as the POST action just queues up an email and updates the status monitor.